docs/releases/3.0.txt

```diff
@@ -54,7 +54,9 @@ Minor features
 :mod:`django.contrib.auth`
 ~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-* ...
+* Set HTML ``autocapitalize`` attribute to ``none`` for the username widget.
+  This prevents the default behaviour of iOS devices of capitalizing
+  the first letter.
```

Comment:
Maybe:
```
* Added HTML ``autocapitalize="none"`` attribute to the widget of username
  field in :mod:`django.contrib.auth.forms`. This prevents automatic
  capitalization, which is the default behavior in some browsers.
```

Reply:
I don't understand why the docs test failed, can't see an error message or anything. Could you explain this to me?

Reply:
You can check errors by clicking for "Docs" details.

`behaviour` -> `behavior` (en-us) and `iOS` doesn't pass spelling check.